mods/cnc/rules/misc.yaml

```diff
@@ -76,6 +76,8 @@ CAMERA:
 	RevealsShroud:
 		Range: 10c0
 		Type: CenterPosition
+	EditorTilesetFilter:
+		Categories: Camera
```

Comment:
Likewise for the other mods, possibly including `flameguy` in TS and the spice blooms in D2K.